creator-node/scripts/updateContentBlacklist.js

```diff
@@ -1,128 +1,292 @@
+  .usage('[action] [type] [ids or cids]')
+  .requiredOption('-t, --type <type>', 'user, track, or cid')
+  .requiredOption('-l, --list <list>', 'comma separated list of ids or cids', ids => ids.split(','))
+  .requiredOption('-a, --act <action>', 'add, remove, or verify')
 
 const { generateTimestampAndSignature } = require('../src/apiSigning')
 
-const PRIVATE_KEY = process.env.delegatePrivateKey
+const PRIVATE_KEY = process.env.delegatePrivateKey // add 0x prefix
 const CREATOR_NODE_ENDPOINT = process.env.creatorNodeEndpoint
 
 // Available action types
-const ACTION_ARR = ['ADD', 'DELETE']
+const ACTION_ARR = ['ADD', 'REMOVE', 'VERIFY']
 const ACTION_SET = new Set(ACTION_ARR)
-const TYPES_ARR = ['USER', 'TRACK']
+const TYPES_ARR = ['USER', 'TRACK', 'CID']
 const TYPES_SET = new Set(TYPES_ARR)
 
 // Script usage:
-// node updateContentBlacklist.js <action> <type> <id>
-// node updateContentBlacklist.js add user 1
-// node updateContentBlacklist.js delete track 4
+// node updateContentBlacklist.js -a add -l 1,3,7 -t user
+// node updateContentBlacklist.js -a add -l 1,3,7 -t track
+// node updateContentBlacklist.js -a add -l Qm..., Qm..., -t cid
+
+// node updateContentBlacklist.js -a remove -l 1,3,7 -t user
+// node updateContentBlacklist.js -a remove -l 1,3,7 -t track
+// node updateContentBlacklist.js -a remove -l Qm..., Qm..., -t cid
+
+// node updateContentBlacklist.js -a verify -l 1,3,7 -t user
+// node updateContentBlacklist.js -a verify -l 1,3,7 -t track
+// node updateContentBlacklist.js -a verify -l Qm..., Qm..., -t cid
+
+// For help:
+// node updateContentBlacklist.js --help
 
 /**
- * Process command line args and either add or delete an entry in/to ContentBlacklist table
+ * Process command line args and either add or remove an entry in/to ContentBlacklist table
  */
 async function run () {
   let args
   try {
     args = parseEnvVarsAndArgs()
   } catch (e) {
-    console.error(`\nIncorrect script usage: ${e.message}`)
-    console.error(`action: [${ACTION_ARR.toString()}]\ntype: [${TYPES_ARR.toString()}]\nid: [integer of 0 or greater]`)
-    console.error(`Script usage: node addToContentBlacklist.js <action> <type> <id>`)
+    console.error(`\nIncorrect script usage: ${e.message}\n`)
+    console.error(`action: [${ACTION_ARR.toString()}]\ntype: [${TYPES_ARR.toString()}]\nids: [list of ints 0 or greater]\ncids: [list of cids]`)
     return
   }
 
-  const { action, type, id } = args
-  // Add or remove type and id entry to ContentBlacklist
+  const { action, type, ids, cids } = args
+
   try {
-    switch (action) {
-      case 'ADD': {
-        await addToContentBlacklist(type, id)
+    switch (type) {
+      case 'USER':
+      case 'TRACK':
+        switch (action) {
+          case 'ADD': {
+            await addIdsToContentBlacklist(type, ids)
+            break
+          }
+          case 'REMOVE': {
+            await removeIdsFromContentBlacklist(type, ids)
+            break
+          }
+          case 'VERIFY': {
```

Comment:
ideally we just run the verification after running the add to blacklist function automatically. like remove the verify case and your switch statement can just be like 

```
switch(action) {
  case 'ADD': {
    await addCIDsToContentBlacklist(cids)
    // new function that fetches each cid from cnode and verify it's been blacklisted
    // in the case of user, fetch https://discoveryprovider3.audius.co/tracks?user_id=<user_id> and for each track pick segment 0 and verify the cnode doesn't return. for track, just fetch https://discoveryprovider3.audius.co/tracks?id=<track_id>. you can make discprov endpoint a query param too
    await verifyCIDsInContentBlacklist(cids)
    break
  }
```